src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -241,6 +241,7 @@ let enableEventPositionChanged = true;
 
 /**
  * globe controls events
+ * @event
  * @property PAN_CHANGED {string} emit after camera pan
```

Comment:
Should those be changed to individual `@event` for each possible event (`GlobeControls#pan-changed`?), each with its own set of `@property` to document the event's properties? (instead of, or in addition to, the examples)

…and those `@property` could possibly be kept to document the _constants_ one could use (instead of the event name as string), but I think `@event` here is wrong.